weave-js/src/core/ops/domain/artifactVersion.ts

```diff
@@ -114,6 +114,22 @@ export const opArtifactVersionSize = makeArtifactVersionOp({
   resolver: ({artifactVersion}) => artifactVersion.size,
 });
 
+export const opArtifactVersionTTLDurationSeconds = makeArtifactVersionOp({
+  name: 'artifactVersion-ttlDurationSeconds',
+  argTypes: artifactVersionArgTypes,
+  description: `Returns the ttl duration in seconds that ${docType(
```

Comment:
Hmm can we clarify this language?  What about `Return the number of seconds remaining until <artifactVersion> is marked for deletion` -- presuming this is actually the case.  Or, is it the original TTL?

Reply:
Its suppose to be the laiter. changed to:
`Returns the original duration in seconds that a ${docType('artifactVersion')} is expected to live for before it gets deleted`